### DISCUSSION TITLE
articles/2024/changing-django51-lookup-labels/

### DISCUSSION THREAD

**Discussion by giscus** [loopwerk/loopwerk.io#18] [Article discussions]
# articles/2024/changing-django51-lookup-labels/

Django 5.1 adds related field lookup to the model admin’s list_display, but with an annoying quirk. Let’s fix that!

https://www.loopwerk.io/articles/2024/changing-django51-lookup-labels/

**Comment by tylerlaprade** [comment]
Love this, thanks again. Another problem I ran into today - if you run your app in production using `gunicorn` rather than `manage.py runserver`, this override won't be applied. I tried putting it in `settings.py`, but like you said in the article, it doesn't get applied in time if it's in there.

**Reply by kevinrenskers** [reply]
Damn, you're absolutely right. I've been testing this locally and didn't think about the deployed site yet. But yeah.. the fix wouldn't apply there. I'll try to find another fix.

**Reply by kevinrenskers** [reply]
For me putting it in the top of `settings.py` does actually work. Tested locally and on the deployed site.

**Reply by tylerlaprade** [reply]
I just tried it again, even sticking it above some of my imports in case one of them had a side effect, but still no luck. I wonder if there's something unique about my project setup preventing it from working in `settings.py`.
